internal/cli/ops_manager_agents_list.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 MongoDB Inc
```

Comment:
[nit] this file should be named `internal/cli/ops_manager_servers_list.go` we (so far) always match the command in our naming